client/src/index.tsx

```diff
@@ -19,4 +19,4 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: http://bit.ly/CRA-PWA
-serviceWorker.register();
+//serviceWorker.register();
```

Comment:
@nurub52, if you want to not have a service worker for now, it is better to just revert it to `serviceWorker.unregister()`. That makes sure that the previous SWs are cleared.

Reply:
updated as commented